cypress/integration/vitals.js

```diff
@@ -0,0 +1,59 @@
+function nthChild(CyElement, Iterator) {
+    if (Iterator < 1 ){
+        return undefined
+    }
+    if (Iterator !== 1) {
+        return nthChild(CyElement.next(), Iterator - 1)
+    }
+    return CyElement.first()
+}
+
+function getNthPointOf(Item, Number) {
+    return nthChild(cy.contains(Item).next().children(),Number)
+}
+
+function checkNumberOfPoints(Item, ExpectedInit, ExpectedFill, Scale) {
+    var CurrentItem=cy.contains(Item).next().children()
+    var i = 1
+    while ('UP DOG'){
```

Comment:
nebolo by lahsie

for i=1; i<scale; i++ ?

miesto while?

Reply:
jak si hovoril, mozes to zkusit...
problem je ze `currentItem next()` mozes urobit len 14x, pri 15-om pouziti hodi test chybu, mozno si treba lepsie nastudovat cypress a da sa to osetrit
jedinu optimalizaciu co som nasiel je ze sa da prec `i` a snizujes ten celkovy pocet, a prestavas na to podmienky (takze je tam o 1 riadok a 1 premennu menej)
zkusal som to prepisat na rekurzivnu funkciu, ale vo vysledku bola na stejne riadkov a v podstate to bol prepis toho `while` bez `i`
neviem co je pre teba citelnejsie, tak som to nechal tak

Reply:
poslal som commit v ktorom je to for cyklus, pridam do slacku video

Reply:
hmm ok, nech sa na to divam ako na while, for, alebo rekurzivnu funkciu... proste sa mi to nepaci
nemam vyhrady k tvojej verzii, je vidiet ze si sa nad tym zamyslel a dobre to upravil
nedal si approve, takze moja otazka je ci je nieco v neporiadku, 
na slacku som video nevidel v char sheetu nic neni ani v DM

Reply:
approve som nedaval ked som tam pridal svoj kod, cakal som co s tym budes robit